tests/test_syst_validation.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Script to test validate_systs.py with test input located in test_syst_validation_input. This script has been tested in the python:3.7 docker container with pyhf, parse, and matplotlib installed
+# Note: this script is assumed to be run from one level above the directory containing the script.
+
+cd tests
```

Comment:
If I run

```
shellcheck tests/*.sh
```

there are a few things that pop up on `tests/test_syst_validation.sh`. Specifically
- https://www.shellcheck.net/wiki/SC1083
- https://www.shellcheck.net/wiki/SC2164

can you take a look at these and either address them, or if you feel they aren't important add the proper ignore pragma (I guess it isn't really a pragma, but the comment thing that acts like a pragma) above them?